tmux/record_ros_bag.yml

```diff
@@ -57,37 +57,40 @@ windows:
   # ROS-TCP-Endpoint
   - sensor_input:
       layout: even-vertical
-      root: ./ros_bags
       panes:
-        - ros_bag_record: ros2 bag record
-            ${ROS_NAMESPACE}/PVFramesBGR
-            ${ROS_NAMESPACE}/HeadsetPoseData
-            ${ROS_NAMESPACE}/HeadsetAudioData
-            ${ROS_NAMESPACE}/HandJointPoseData
-            ${ROS_NAMESPACE}/SpatialMapData
-            ${ROS_NAMESPACE}/EyeGazeData
-            ${ROS_NAMESPACE}/AnnotationEvents
-            ${ROS_NAMESPACE}/HeadsetDepthPoseData
-            ${ROS_NAMESPACE}/ShortThrowDepthMapImages
-        #- datahub: ros2 run ros_tcp_endpoint default_server_endpoint --ros-args
-        #    -r __ns:=${ROS_NAMESPACE}
-        #    -p ROS_IP:=0.0.0.0
+        - datahub: ros2 run ros_tcp_endpoint default_server_endpoint --ros-args
+            -r __ns:=${ROS_NAMESPACE}
+            -p ROS_IP:=0.0.0.0
         - hl2ss_bridge: ros2 run angel_system_nodes hl2ss_ros_bridge --ros-args
             -r __ns:=${ROS_NAMESPACE}
             -p ip_addr:=${HL2_IP}
             -p image_topic:=PVFramesBGR
             -p hand_pose_topic:=HandJointPoseData
             -p audio_topic:=HeadsetAudioData
-            -p sm_topic:=SpatialMapData
-            -p head_pose_topic:=HeadsetPoseData
+            -p sm_topic:=disable
+            -p head_pose_topic:=disable
```

Comment:
Does head pose topic also need to be disabled?

Reply:
Just updated, seems to work